node/bridge_test.go

```diff
@@ -99,7 +98,6 @@ func TestBridge_P2P_Streams(t *testing.T) {
 
 func TestBridge_WithRemoteCore(t *testing.T) {
 	// TODO(@Wondertan): Fix core
-	t.Skip("Skip until we fix core")
```

Comment:
what does "fix core" mean @Wondertan

Reply:
Fix core means update to the latest Tendermint

Reply:
There was an issue in old Tendermint that could panic occasionally.

Reply:
It was panicking only on CI though, so we need to check how it goes again, as we now rely on the latest changes.